lib/sidekiq/testing.rb

```diff
@@ -143,28 +202,36 @@ module Worker
     #
     module ClassMethods
 
+      # Queue for this worker
+      def queue
+        self.sidekiq_options["queue"]
```

Comment:
I updated the test to specify a symbol queue and all still passed, so that confirms and should ensure that condition is tested going forward.